packages/firebase_messaging/README.md

```diff
@@ -54,7 +54,74 @@ Note: When you are debugging on Android, use a device or AVD with Google Play se
       <category android:name="android.intent.category.DEFAULT" />
   </intent-filter>
   ```
+#### Optionally handle background messages
 
+By default background messaging is not enabled. To handle messages in the background:
+
+1. Add an Application.java class to your app
```

Comment:
Where in the App we should put this Java class, with the MainActivity or inside io.flutter.plugins?

Reply:
@JillyTaboga You should add it alongside your `MainActivity.java` file.